.github/bin/dist-nuget.sh

```diff
@@ -30,7 +30,10 @@ if [ "$GITHUB_REPOSITORY" != "planetarium/libplanet" ] || (
 fi
 
 package_version="$(cat obj/package_version.txt)"
-dotnet nuget push \
-  "./$project/bin/$configuration/$project.$package_version.nupkg" \
-  --api-key "$NUGET_API_KEY" \
-  --source https://api.nuget.org/v3/index.json
+
+for project in $projects; do
```

Comment:
Using the proper array:

```suggestion
for project in "${projects[@]}"; do
```